<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerWindow</class>
 <widget class="QDialog" name="ComputerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Computer</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>230</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_computer">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label_new_person">
      <property name="text">
       <string>&lt;font size=&quot;5&quot;&gt;New Computer&lt;/font&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_computer_name">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_computer_type">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_computer_date">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_computer_ifBuilt">
          <property name="text">
           <string>Was it built:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_computer_name">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_name_error_star">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_computer_type">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_type_error_star">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QDateEdit" name="dateEdit_year_built">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>24</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="maximumDate">
             <date>
              <year>7920</year>
              <month>12</month>
              <day>31</day>
             </date>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::YearSection</enum>
            </property>
            <property name="displayFormat">
             <string notr="true">yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_date_error_star">
            <property name="text">
             <string>&lt;font color=&quot;white&quot;&gt; Error &lt;/font&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radioButton_yes">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_no">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_built_error_star">
            <property name="text">
             <string>&lt;font color=&quot;white&quot;&gt; Error &lt;/font&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_computer_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_computer_addRelation">
        <property name="text">
         <string>Add relation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_computer_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_computer_error">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;font color='red'&gt;*All textboxes and buttons must be filled in&lt;br&gt;or checked&lt;/font&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
